npm-update

```diff
@@ -95,6 +104,8 @@ def run(specified_package, verbose=False, **kwargs):
         return "~" + version
 
     for package in packages:
+        if package in pending_updates:
```

Comment:
Would be nice to get a log message like "Already pending in PR #1234" here, so that it's easier to debug when running manually. For that, `pending_updates` should be a `package → PR` map.

Reply:
Right, I think it is more useful to list all ignored packages due to existing PR than just saying it on the one, that was randomly picked from the upgradeable packages.

Now it shows message like:
```
Ignoring 'qunit' as there is pending PR #14841
```
for every existing PR.